modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/ConfigurationContainer.tsx

```diff
@@ -23,65 +23,70 @@ export function ConfigurationContainer({
 		: values.system;
 
 	return (
-		<ClayPanel
-			collapsable
-			defaultExpanded
-			displayTitle={Liferay.Language.get('configuration')}
-			displayType="unstyled"
-		>
-			<ClayPanel.Body>
-				<div className="lfr-objects__object-definition-details-configuration">
-					<Toggle
-						disabled={
-							isReadOnly || !hasUpdateObjectDefinitionPermission
-						}
-						label={Liferay.Language.get('show-widget')}
-						name="showWidget"
-						onToggle={() => setValues({portlet: !values.portlet})}
-						toggled={values.portlet}
-					/>
+		<>
```

Comment:
Hey @igor-franca, since we have a `div` enclosing all the other components, we don't need that fragment, you can remove it.